app/login/page.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import Image from 'next/image';
+
+export default function LoginPage() {
+  return (
+    <div className="flex-1 flex">
+      <main className="flex-1 flex items-center justify-center p-7">
+        <div className="w-full max-w-md p-8 rounded-lg border">
+          <h2 className="text-2xl font-bold text-center mb-4">로그인</h2>
+          <p className="text-center text-gray-600 mb-8">
```

Comment:
```suggestion
          <p className="text-center text-gray-400 mb-8">
```
글씨가 피그마 상에서랑 색이 다른거 같아여

Reply:
헉 수정하면서 좀 바뀌었나봐요 잘 확인했어야했는데 ㅜㅜ.. 감사합니다 ㅠㅠ!!